src/Bonnier/Admin/OAuth.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Ignore this class as there is a seperate pull request for this feature (this one is ooooold)